sky/provision/kubernetes/utils.py

```diff
@@ -1336,13 +1336,20 @@ def check_credentials(context: Optional[str],
         return False, ('An error occurred: '
                        f'{common_utils.format_exception(e, use_bracket=True)}')
 
+    # Check if $KUBECONFIG envvar consists of multiple paths. We run this before
+    # optional checks.
+    kubeconfig_path = _get_kubeconfig_path()
+    if len(kubeconfig_path.split(os.pathsep)) > 1:
```

Comment:
Looks like this code block is repeated, should we move it inside _get_kubeconfig_path()?